src/vivarium/framework/results/interface.py

```diff
@@ -73,6 +73,7 @@ def register_stratification(
         self,
         name: str,
         categories: List[str],
+        excluded_categories: Optional[List[str]] = None,
```

Comment:
Per comment above, do we just convert the dict from the config into a list so now these will always be a list?

Reply:
This is at the stratfiication-level and so is always a list. The model spec also provides a list at the stratification level, it's just that each stratification is stored as the keys to a dictionary.